README.md

```diff
@@ -1,24 +1,41 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Platform Console: for creating and deploying applications on the VA Platform.
 
-Things you may want to cover:
+## Getting Started
 
-* Ruby version
+From the root of the repository, run:
 
-* System dependencies
+```bash
+brew install postgres
+brew services start postgresql
+bin/setup
+bin/dev
+```
 
-* Configuration
+This will:
+1. Install and run Postgres.
+2. Install the necessary gems, create the pg database, and seed the database.
+3. Start the Rails server and Tailwind watcher.
 
-* Database creation
+This has been tested on macOS Monterey using Ruby 3.1 and PG 14.1.
 
-* Database initialization
+## Running tests
 
-* How to run the test suite
```

Comment:
Do you want to specify that we are using minitest vs rspec? I know this is probably obvious to the developer, but we may want to explicitly call it out. I also remember talking about using minitest with rspec syntax. Did we want to move forward with that?

Reply:
I think if they want to look at our stack they can jump into the Gemfile. I didn't mention a lot of things here as the Gemfile is well documented (e.g. Stimulus, flipper, Sidekiq, Redis).

We can use whatever we'd like (minitest, minitest spec, or rspec). I've only added a minimal amount of tests using the Rails default (minitest) to keep coverage up.

Reply:
Ok sounds good to me! Merge away 👏